lib/event_sources.json.example

```diff
@@ -11,7 +11,8 @@
     {
       "ScheduleName": "node-lambda-test-schedule",
       "ScheduleState": "ENABLED",
-      "ScheduleExpression": "rate(1 hour)"
+      "ScheduleExpression": "rate(1 hour)",
+      "ScheduleDescription": "Run node-lambda-test-function once per hour"
```

Comment:
if the default `${params.ScheduleName} - ${params.ScheduleExpression}` is "enough" in 99% of the cases, it doesn't make sense to me to set this here and overwrite it for all functions.

Would it make more sense to not have this value here, but simply document it?